index.test.js

```diff
@@ -0,0 +1,74 @@
+const http = require('http');
+const supertest = require('supertest');
+const handler = require('./src/handler');
```

Comment:
Let's load the handler from `index.js`, since that's the file this test is co-located with.